userguide/storage.rst

```diff
@@ -240,15 +239,21 @@ intended use:
   volume (pool) is created on top of the encrypted devices. Data is
   encrypted as it is written and decrypted as it is read.
 
-* This type of encryption is primarily useful for users storing
-  sensitive data but wanting the ability to remove disks from the pool
-  without having to first wipe the disk contents.
-
-* The %brand% encryption design is only suitable for safe disposal of
-  disks independent of the encryption key. As long as the key and the
-  disks are intact, the system is vulnerable to being decrypted.
-  Protect the key with a strong passphrase and secure all backups of
-  it.
+* The type of encryption used by %brand% is primarily used to protect 
+  data against the risks of disks (or the entire system) being physically
+  stolen, or removed to view their contents when the system is turned off.
```

Comment:
There is a weird break with "a system being stolen or removed".  The comma does not help, making the sentence longer but not clearer.

Reply:
rephrased - with added serial commas!